.github/workflows/cmake.yml

```diff
@@ -175,6 +175,12 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
+    - name: Information about platform
+      run: |
+        sycl-ls
```

Comment:
sycl-ls is only available after you download dpcpp. I think we might do this step at the very end of the job (but it has to run with `if: always()` then)